src/PlusOpenIGTLink/PlusIgtlClientInfo.cxx

```diff
@@ -53,12 +54,19 @@ PlusStatus PlusIgtlClientInfo::SetClientInfoFromXmlData(vtkXMLDataElement* xmlda
 
   if (xmldata->GetAttribute("TDATARequested") != NULL)
   {
-    clientInfo.TDATARequested = STRCASECMP(xmldata->GetAttribute("TDATARequested"), "TRUE") == 0;
+    clientInfo.SetTDATARequested(STRCASECMP(xmldata->GetAttribute("TDATARequested"), "TRUE") == 0);
   }
 
   if (xmldata->GetAttribute("Resolution") != NULL)

```

Comment:
Is there a reason not using the usual config XML macros?

Reply:
No, just updated existing code. Will convert to macros.